app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import json
+from decimal import Decimal
+
+
+def calculate_profit(name: str):
+    with open(name) as f:
+        data = json.load(f)
+        bought = 0
+        sold = 0
+        bought_account = 0
+        sold_account = 0
+        for trans in data:
+            if trans["bought"] is not None:
+                bought += Decimal(trans["bought"])
+                bought_account += \
```

Comment:
Try not to use `\` for line continuation. Read [pep8](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) to know how to move operators to a new line.

Reply:
but flake8 doesn't like move like those, which said in pep8

Reply:
https://prnt.sc/ZPOrxhBZH-0Z